src/agent.rs

```diff
@@ -92,7 +92,7 @@ pub trait Dispatched: Agent + Sized + 'static {
     /// bridge around if you wish to use a dispatcher. If you are using agents in a write-only manner,
     /// then it is suggested that you create a bridge that handles no-op responses as high up in the
     /// component hierarchy as possible - oftentimes the root component for simplicity's sake.
-    fn dispatcher() -> Box<dyn Dispatcher<Self>>;
+    fn dispatcher() -> Box<dyn Bridge<Self>>;
```

Comment:
I'm in favor of creating a distinct types for Bridges and Dispatchers. I feel like the distinct types communicate that the object they got from `dispatcher` behaves differently than one returned from `bridge`.

Maybe newtyping like `DispatcherHandle(Box<dyn Bridge>)` to indicate that it is distinct might be worth doing? That would make the API look funky with one bridgelike function returning a trait object, and the other a newtype of a trait object. Some breaking change might be needed down the line to make those behave similarly (with something like `BridgeHandle(Box<dyn Bridge>)`.

Reply:
I'm going to merge this, and add the newtype around the response for bridge. Thanks for simplifying the implementation.